Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM python:3.10.3-slim-buster as base
+RUN apt-get update
+COPY . /opt/
+WORKDIR /opt
+RUN pip install -r requirements.txt
```

Comment:
What's your plan for the future? Will you manually keep requirements.txt and pyproject.toml in sync?

Your project uses Poetry (which is more fully featured than just pip). There is never any need to run pip commands yourself - just run `poetry install`.

And delete the requirements.txt file.

If you don't want a virtual environment inside the image (fair enough - it doesn't need one), then you can run `poetry config virtualenvs.create false --local && poetry install`. This will turn off virtual environment creation before running the install.